x-pack/plugins/fleet/public/applications/fleet/sections/agents/components/agent_upgrade_modal/constants.tsx

```diff
@@ -9,6 +9,11 @@ export const ROLLING_UPGRADE_MINIMUM_SUPPORTED_VERSION = '8.3.0';
 
 // Available versions for the upgrade of the Elastic Agent
 export const FALLBACK_VERSIONS = [
+  '8.3.2',
```

Comment:
what about 8.4.0?

Reply:
The code dynamically adds the current version, so 8.4.0 will be there. However I'm working on https://github.com/elastic/kibana/issues/133309 that will make the hardcoded list less relevant.